community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/suspend.py

```diff
@@ -30,6 +29,7 @@
 )
 from util import lookup
 import tpu
+import mig_flex
```

Comment:
Maybe I should get some new glasses but where is this coming from? Is this some package or a python file of some sort?

Reply:
Forget to `git add`